app/models/user_stat.rb

```diff
@@ -177,6 +177,21 @@ def self.update_view_counts(last_seen = 1.hour.ago)
        WHERE X.user_id = user_stats.user_id AND
              X.c <> posts_read_count
     SQL
+
+    # Update denormalized post_edits_count
```

Comment:
I'm curious about this - if it's updated in post revisor why would we also need to update the denormalized counts here?

Reply:
Agree with @eviltrout here, I don't think we need to correct the counts here for users on an hourly basis.

Reply:
I Will remove it, you're right, no need to update here.